spt/utils/game_detection.cpp

```diff
@@ -128,6 +128,25 @@ namespace utils
 		return result;
 	}
 
+	const char* GetGameName()
+	{
+		if (DoesGameLookLikePortal())
+			return "Portal 1";
```

Comment:
It's "Portal", not "Portal 1". Saying Portal 1 implies Portal 2 exists and I hate Portal 2.

Reply:
good for you